prov/rxm/src/rxm_ep.c

```diff
@@ -1138,7 +1138,7 @@ static int rxm_ep_msg_cq_open(struct rxm_ep *rxm_ep)
 		return ret;
 	}
 
-	if (cq_attr.wait_obj != FI_WAIT_FD)
+	if ((cq_attr.wait_obj != FI_WAIT_FD))// || domain->passthru)
```

Comment:
The domain->passthru part is commented out. Does it need to be uncommented or removed?

Reply:
This was a debug hack and will be removed in the next push.